nodejs/week1/avg.js

```diff
@@ -0,0 +1,22 @@
+// Program for calculate the average of the numbers provided as command line arguments.
+
+let total = 0;
+let count = 0;
+let average;
+
+for (let i = 2; i < process.argv.length; i++) {
+  let num = Number(process.argv[i]);
+  if (isNaN(num)) {
+    console.log(`All the arguments should be a number`);
+    return;
+  } else {
+    total += num;
+    count++;
+  }
+}
```

Comment:
this looks great, how about trying to do the same using array functions? :)

Reply:
Thank you for your review. Array function used.